man/cide.1.md

```diff
@@ -91,6 +101,17 @@ https://docs.docker.com/docker/reference/commandline/cli/#environment-variables:
 *DOCKER_TLS_VERIFY*
   When set Docker uses TLS and verifies the remote.
 
+
+The following environment variables are used by the Amazon `s3` SDK to
+authenticate, and should be set by the user. See also
+http://docs.aws.amazon.com/sdkforruby/api/index.html.
+
+*S3_USER*
+  The Amazon s3 username
+
+*S3 PASS*
+  The amazon s3 pass
```

Comment:
The following environment variables are used by the AWS S3 uploader to authenticate. See also
 +http://docs.aws.amazon.com/sdkforruby/api/index.html.

AWS_ACCESS_KEY_ID, AWS_SECRET_ACCESS_KEY, AWS_REGION

we also use the AWS_BUCKET to select which S3 bucket to upload the package to